### ISSUE TITLE
Web contents  invisible when keyboard show in Android [flutter_webview]

### ISSUE THREAD

**Issue by CarGuo** [flutter/flutter#51254]
## Steps to Reproduce

1. Open WebView with  ` https://github.com/login ` or `https://github.com/login/oauth/authorize?client_id=xxxxxx&state=app&redirect_uri=xxxxxx`.
2. Tap Username textInput show keyboard , then hide keyboard .
3. Tap Password textInput show keyboard, then web contents  invisible .
4.  As under shown gif :

![4444444](https://user-images.githubusercontent.com/10770362/75090033-bdbf2780-5599-11ea-9e52-eb167b39b13c.gif)

## ENV

**This problem may be only appears in Huawei** :

- webview_flutter: ^0.3.19+8
- HUAWEI P30pro - Android 10 / HUAWEI P20pro - Android 9.0 (bad)
- Redmi 5 plus - Android 8.1.0  (not reproduce)
- simulator API 29  (not reproduce)

## Log

```
W/cr_AwAutofillManager(27666): WebView autofill is disabled because WebView isn't created with activity context.
I/InputMethodManager(27666): showSoftInput
I/InputMethodManager(27666): mServedView =com.shuyu.gsygithub.gsygithubappflutter;view =com.shuyu.gsygithub.gsygithubappflutter;flags =0
W/InputMethodManager(27666): startInputReason = 3
I/ViewRootImpl(27666): jank_removeInvalidNode all the node in jank list is out of time
I/ViewRootImpl(27666): jank_removeInvalidNode all the node in jank list is out of time
W/cr_AwAutofillManager(27666): WebView autofill is disabled because WebView isn't created with activity context.
W/InputMethodManager(27666): startInputReason = 3
I/InputMethodManager(27666): showSoftInput
I/InputMethodManager(27666): mServedView =com.shuyu.gsygithub.gsygithubappflutter;view =com.shuyu.gsygithub.gsygithubappflutter;flags =0
```

## Doctor

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.12.13+hotfix.8, on Mac OS X 10.15.3 19D76, locale
    zh-Hans)
 
[!] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor
      --android-licenses
[✓] Xcode - develop for iOS and macOS (Xcode 11.3)
[✓] Android Studio (version 3.5)
[!] IntelliJ IDEA Ultimate Edition (version 2018.3.5)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[✓] Connected device (3 available)

! Doctor found issues in 2 categories.
```

**Comment by wurstnudl** [comment]
Please remove the device specific label. It doesn't work ony any android device, including emulators.
Also tried various different sites.

Sometimes, there's a suspicious log output:

`W/IInputConnectionWrapper(18389): getSelectedText on inactive InputConnection
W/IInputConnectionWrapper(18389): getTextAfterCursor on inactive InputConnection`

Might be related to https://github.com/flutter/flutter/issues/36478?

It works fine on ios.

**Comment by 0-php** [comment]
i have this question too

**Comment by blasten** [comment]
This has been fixed in Flutter 1.22/ WebView 1.0 - use hybrid composition in the Flutter WebView plugin.

https://github.com/flutter/plugins/tree/master/packages/webview_flutter#using-hybrid-composition
